content/news/draft-bevy-0.14/index.md

```diff
@@ -36,6 +36,7 @@ Peering deep into the mists of time (predictions are *extra* hard when your team
 - **Curve abstraction:** Curves come up all of the time in game dev, and the mathmagicians that make up the [Curve Crew](https://discord.com/channels/691052431525675048/1236110755212820581) are [designing a trait](https://github.com/bevyengine/rfcs/pull/80) to unify and power them.
 - **Better text:** our existing text solution isn't up to the demands of modern UI. We're looking at replacing it with a better solution.
 - **A unified view on dev tools:** In 0.14, we've added a stub `bevy_dev_tools` crate: a place for tools and overlays that speed up game development such as performance monitors, fly cameras, or in-game commands to spawn game objects. We're working on adding more tools, and creating a [dev tool abstraction](https://github.com/bevyengine/rfcs/pull/77). This will give us a unified way to enable/disable, customise and group this grab bag of tools into toolboxes to create something like Quake console or VSCode Command Palette with tools from around the ecosystem.
+- **A modular, maintainable render graph:** Bevy's existing rendering architecture is already quite good at providing reusable renderer features such as RenderPhases, batching, and draw commands. However, the render graph itself is one area that is greatly lacking. While the exact design hasn't been finalized (and feedback is greatly encouraged!!!), we've been actively working to [redesign the render graph](https://github.com/bevyengine/bevy/pull/13397) in order to build up to a larger refactor of the renderer towards modularity and ease of use.
```

Comment:
I'd like to hear more about why the render graph is currently bad in here: another half-sentence or so is all that's needed.

Reply:
done!